Makefile.am

```diff
@@ -22,23 +22,35 @@ noinst_LIBRARIES =
 lib_LTLIBRARIES =
 module_LTLIBRARIES =
 pkginclude_HEADERS =
+nodist_pkginclude_HEADERS =
 dist_examples_DATA =
 
 include lib/subdir.am
 include zebra/subdir.am
 include qpb/subdir.am
 include fpm/subdir.am
 
+include ripd/subdir.am
+include ripngd/subdir.am
+include ospfd/subdir.am
+include ospf6d/subdir.am
+include ospfclient/subdir.am
+include isisd/subdir.am
+include nhrpd/subdir.am
+include ldpd/subdir.am
+include babeld/subdir.am
+include eigrpd/subdir.am
+
 SUBDIRS = . @LIBRFP@ @RFPTEST@ \
-	 @BGPD@ @RIPD@ @RIPNGD@ @OSPFD@ @OSPF6D@ @LDPD@ \
-         @ISISD@ @PIMD@ @NHRPD@ @EIGRPD@ @BABELD@ \
-	 @WATCHFRR@ @VTYSH@ @OSPFCLIENT@ @DOC@ \
+	 @BGPD@ \
+         @PIMD@ \
```

Comment:
i'm chugging through the remaining ones either way, removing them & replacing them with non-recursive ones, so i didn't feel it worth to fix the preexisting murked indentation...